app/views/install/compose.phtml

```diff
@@ -221,6 +221,34 @@ services:
       - _APP_DB_USER
       - _APP_DB_PASS
 
+  appwrite-worker-builds:
+    image: <?php echo $organization; ?>/<?php echo $image; ?>:<?php echo $version."\n"; ?>
+    entrypoint: worker-builds
+    container_name: appwrite-worker-builds
+    restart: unless-stopped
+    networks:
+      - appwrite
+    volumes: 
+      - ./app:/usr/src/code/app
+      - ./src:/usr/src/code/src
```

Comment:
In this tempalte there shouldn't be volume mounts 👍🏻

Reply:
Fixed 👍